src/db/tripledb.plt

```diff
@@ -110,5 +110,19 @@ test('tripledb tell triple with Unit'):-
     assert_true(tripledb_tell(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter'))),
     assert_true(tripledb_ask(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter'))).
 
+% test for special characters in iri: @*~!#?
+test('tripledb_tell_special_characters_@*~!#?'):-
+    tripledb_tell(test_datatype:'normal_user_test_new', test_datatype:'last@*~!#?_name', 'umlaut').
+
+% it is observed that once these special characters are stored in db(with wrong format), they can not be retrieved normally in variables or any other way.
```

Comment:
that does not seem reasonable to me to be intended behaviour. It rather seems like a BUG that you can assert but that you cannot retrieve again

Reply:
well it is due to encoding issues we have discussed before. If you think it should fail the storing this sort of strings in first place then we can think about creating new issue for this task and can discuss how to resolve this problem there.

Reply:
yes I know. But the example in the test is very valid. And it seems to me that it would be bad to not support it. Sorry for the confusion! I mean, umlaut in IRIs is one thing, but it must be ok to use them as values for datatype properties.
Seems I have to revisit encoding.

For now, could you change the *assert_false* to *assert_true*, and disable the test with fixme option?